src/index.html

```diff
@@ -9,10 +12,65 @@
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="styles/style.scss"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
     />
   </head>
-  <body>
-    <h1>Product cards</h1>
+
+  <body class="page__body">
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <a class="card__image">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="imac"
+        />
+      </a>
+
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina
+        <br />
+        5K Monoblock (MNED2UA/A)
+      </h2>
+      <p class="card__code">Product code: 195434</p>
+
+      <div class="card__rating">
+        <div class="card__rating-stars stars">
```

Comment:
Add the modifier classes
```suggestion
        <div class="stars stars--4">
```